configs/dynamic-plugins/dynamic-plugins.override.example.yaml

```
@@ -4,6 +4,16 @@
 includes:
   - dynamic-plugins.default.yaml
 
+# Functioning example, uncomment to use
+# plugins:
+  #Tech Radar frontend plugin
+  # - package: ./dynamic-plugins/dist/backstage-community-plugin-tech-radar
+  #   disabled: false
+
+  # Tech Radar backend plugin
```

```suggestion
  # Tech Radar backend plugin. Required if you want to customize the tech radar URL (see the `techRadar.url` app-config property).
  # Find out more in https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.7/html-single/customizing_red_hat_developer_hub/index#proc-customizing-the-tech-radar-page_customizing-the-quickstarts
```

I'd add more details about this..

---

Thanks, Armel. Have made those changes now.